src/main/java/org/kpmp/packages/Package.java

```diff
@@ -49,6 +50,10 @@ public String getPackageType() {
 	}
 
     public String getStudy(){
+        if (Objects.equals(study, "CureGN Diabetes")){
```

Comment:
I'm concerned that, down-the-line, if we ever need to get the study name for something else it's going to be this name without spaces. Maybe try making a new method called "getStudyFolderName" or something like that which usually returns "getStudy" but returns this special string if it's CureGN.